libs/partners/openai/langchain_openai/chat_models/azure.py

```diff
@@ -201,7 +201,10 @@ def validate_environment(cls, values: Dict) -> Dict:
     @property
     def _identifying_params(self) -> Dict[str, Any]:
         """Get the identifying parameters."""
-        return {**self._default_params}
+        return {
+            **{"deployment_name": self.deployment_name},
```

Comment:
should we call this azure_deployment, since thats the param name in openai v1 sdk  (deployment_name was openai v0 name that we keep supporting for backwards compatibility)\
```python
**{"azure_deployment": self.deployment_name},
```

Reply:
> should we call this azure_deployment, since thats the param name in openai v1 sdk (deployment_name was openai v0 name that we keep supporting for backwards compatibility)\
> 
> ```python
> **{"azure_deployment": self.deployment_name},
> ```

Thanks for code review, I've fixed it

Reply:
> > should we call this azure_deployment, since thats the param name in openai v1 sdk (deployment_name was openai v0 name that we keep supporting for backwards compatibility)\
> > ```python
> > **{"azure_deployment": self.deployment_name},
> > ```
> 
> Thanks for code review, I've fixed it

@baskaryan PTAL